readme.md

```diff
@@ -1,4 +1,4 @@
-<img height="200" src="https://raw.githubusercontent.com/cytomining/cytotable/main/logo/with-text-for-light-bg.svg?raw=true">
+<img height="200" src="https://raw.githubusercontent.com/cytomining/cytotable/main/logo/with-text-for-light-bg.png?raw=true">
```

Comment:
Very optional (and slightly tangential to the PR focus): consider using the directory under `docs/source/_static` for storing the logo images (matching the `dataflow` markdown image below) and related reference within the root readme to match. Sphinx recognizes the `_static` dir as a common media location when rendering the docs. This would help keep the documentation images in the same spot.

Note: if you opt for this, it might be that the `docs/source/_static/logo/README.md` file within the logo dir needs to be added to the conf.py `excludes_patterns` list so it isn't referenced through Myst builds.

Reply:
I think there's a give and take with this issue - on the one hand, it seems like an elegant solution, on the other, it changes the way a user may explore the repo for logo images. The latter is likely a super minor thing that has a negligible drawback, but if we make this switch, we should do it consistently for all repos in `cytomining` to create consistent expectation.

I created a new issue to tackle this, as this multi-repo change, pushes this out of scope for this quick fix.

Thanks for the comment!